### ISSUE TITLE
Feature request: additional conductance term for bare soil + bucket model.

### ISSUE THREAD

**Issue by akshaysridhar** [CliMA/SurfaceFluxes.jl#85]
- [ ] Add equations to docs
- [ ] Feature implementation
- [ ] Tests
Reference: (Textbook) Bonan Ch 7 (Equation 7.6, 7.7)
@kmdeck

**Comment by kmdeck** [comment]
thanks Akshay! I'll be able to start work on this next week.

**Comment by kmdeck** [comment]
lol at my earlier comment "starting next week!" from Jan 17.

More realistically, I plan to do this this sprint! (assuming the level of change is similar to that require for the beta factor).

@akshaysridhar

**Comment by tapios** [comment]
Is this still relevant? (@kmdeck)

**Comment by kmdeck** [comment]
I dont think so @tapios . This was talking about adding another argument to the SF interface for computing fluxes, similar to the beta factor, to adjust fluxes. It would have made the SF interface "depend" on the surface model parameterization for T_surface, q_surface.

The new interface should take care of it (we can pass in a parameterization function for T_surface, q_surface, and those encapsulate the additional resistances, or e.g. we could pass in other parameterization functions which model the surface variables in different ways. this new interface should be more flexible in terms of surface model parametrization choices, e.g. the same interface works for beta type models or resistance type models).
